cmd/utils.go

```diff
@@ -205,3 +227,151 @@ func checkLatestVersion() {
 		fmt.Println("\n-------------------------------------------------")
 	}
 }
+
+// Code for generating zsh bash completion script
+// Inspired by https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/completion/completion.go
```

Comment:
I think we should have this similar to how we did in core-scaling: https://github.com/qlik-oss/core-scaling/blob/master/LICENSE